codesec-agent/values.yaml

```diff
@@ -34,6 +34,7 @@ ssl:
 
 proxy:
   url:
+  preference: outbound
```

Comment:
1. Let's add a comment here also to explain the available options. (I know it's already in the README but as a user it's usually added also to the values.yaml.
2. Why default is set to outbound and not both?

Reply:
1. Please explain explicitly the difference between outbound / inbound / both (most convenient is in values.yaml with multi-line comment).
2. Default = outbound makes sense, you can keep it